libavpipe/src/avpipe_xc.c

```diff
@@ -3252,7 +3295,7 @@ skip_for_sync(
     if (input_packet->stream_index == decoder_context->video_stream_index) {
         /* first_key_frame_pts points to first video key frame. */
         if (decoder_context->first_key_frame_pts == AV_NOPTS_VALUE &&
-            input_packet->flags == AV_PKT_FLAG_KEY) {
+            input_packet->flags & AV_PKT_FLAG_KEY) {
```

Comment:
Any reason for changing this condition? Have you seen a case that causes a problem with == condition?